e2e/test_bootstrap_multiple_versions.sh

```
@@ -7,18 +7,21 @@
 SCRIPTDIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
 source "$SCRIPTDIR/common.sh"
 
-# Create constraints file to pin build dependencies (keeps CI fast)
+# Create constraints file with generous ranges to test multiple versions
+# of build dependencies (not just top-level packages)
 constraints_file=$(mktemp)
 trap "rm -f $constraints_file" EXIT
```

Shellcheck (0.11.0)

[warning] Use single quotes, otherwise this expands now rather than when signalled.

(SC2064)